[[source]]
url = "https://pypi.org/simple"
verify_ssl = true
name = "pypi"

[packages]
Babel = ">=2.4.0"
Flask-BabelEx = ">=0.9.3"
invenio = { version = "==3.2.1", extras = ["base", "auth", "metadata", "files", "postgresql", "elasticsearch7" ]}
uwsgi = ">=2.0"
uwsgitop = ">=0.11"
uwsgi-tools = ">=1.1.1"
lxml = ">=3.5.0,<4.2.6"

[dev-packages]
Flask-Debugtoolbar = ">=0.10.1"
Sphinx = ">=1.5.1"
check-manifest = ">=0.35"
coverage = ">=4.4.1"
isort = ">=4.3"
mock = ">=2.0.0"
marshmallow = ">=2.15.1,<3.0.0"
pydocstyle = ">=2.0.0"
pytest = ">=3.3.1"
pytest-cov = ">=2.5.1"
pytest-invenio = ">=1.0.2,<1.4.0"
pytest-mock = ">=1.6.0"
pytest-pep8 = ">=1.0.6"
pytest-random-order = ">=0.5.4"
pytest-runner = ">=3.0.0,<5"
# FIXME: Remove this once this issue is resolved https://github.com/pypa/pipenv/issues/3865.
docutils = "==0.15"

[requires]
python_version = "3.6"

[scripts]
test = "python setup.py test"
build_sphinx = "python setup.py build_sphinx"
